src/components/blockHelpers.tsx

```diff
@@ -365,7 +373,11 @@ export const ContentWrapper: React.FC<ContentWrapperProps> = ({
     }}
   >
     {({ isVisible }) => (
-      <ContentWrapperStyled visible={index <= 1 || isVisible} {...props}>
+      <ContentWrapperStyled
+        visible={index <= 1 || isVisible}
+        brandPivot
```

Comment:
`brandPivot={brandPivot}` ?

Reply:
Do we have a prefer to write the longer boolean syntax? :)

Reply:
Yep, of course you need to do that when you pass down a prop, what was I thinking 😆